### ISSUE TITLE
Unable to create cluster following tuturial

### ISSUE THREAD

**Issue by TaurusUY** [xetys/hetzner-kube#141]
I am following the tutorial step by step but when try to create the kubernetes cluster it fails with the following error and exit code:

I do have: 
Hetzner cloud account
kubectl / kubeadm / kubelet (I use it to manage other clusters running in Rancher / AWS / Minikube)

Versions in my local machine:
_USER@LX-HOST:~h$ **kubeadm version**
kubeadm version: &version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.0", GitCommit:"XXXXXXXXXX", GitTreeState:"clean", BuildDate:"2018-03-26T16:44:10Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/amd64"}

USER@LX-HOST:~$ **kubectl version**
Client Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.2", GitCommit:"XXXXXXXXXX", GitTreeState:"clean", BuildDate:"2018-04-27T09:22:21Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"10+", GitVersion:"v1.10.0-rancher1", GitCommit:"XXXXXXXXXX", GitTreeState:"clean", BuildDate:"2018-03-29T00:13:11Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/amd64"}_

**Cloud instances are been created, but cluster init fails as showed below:**

**Error Output:**

USER@LX-HOST:~$ **hetzner-kube cluster create -d --name kube-test --ssh-key kube-test**
2018/06/06 16:24:22 creating master nodes...
2018/06/06 16:24:23 creating server 'kube-test-master-01'...
  17s [====================================================================] 100%
2018/06/06 16:24:42 Created node 'kube-test-master-01' with IP XXX.XXX.XXX.XXX
2018/06/06 16:24:42 creating server 'kube-test-worker-01'...
  11s [====================================================================] 100%
2018/06/06 16:24:55 Created node 'kube-test-worker-01' with IP XXX.XXX.XXX.XXX
2018/06/06 16:24:55 sleep for 30s...
kube-test-master-01: kubeadm init                               44.4% [=============================>--------------------------------------]
kube-test-worker-01: waiting for master                         50.0% [=================================>----------------------------------]
**2018/06/06 16:25:30 bash: kubeadm: command not found**

2018/06/06 16:25:30 > kubeadm reset && kubeadm init --pod-network-cidr=10.244.0.0/16
2018/06/06 16:25:30 
2018/06/06 16:25:30 
**2018/06/06 16:25:30 Run failed:Process exited with status 127**

I am missing to declare some variable for kubeadm somewhere?

Regards
Alvaro

**Comment by JohnnyQQQQ** [comment]
Are you using the current master?

**Comment by JohnnyQQQQ** [comment]
Did you build the binary from the current master branch or did you download
the binary from a release?

Alvaro <notifications@github.com> schrieb am Do., 7. Juni 2018, 08:25:

**Comment by JohnnyQQQQ** [comment]
Did you build the binary from the current master branch or did you download
the binary from a release?

**Comment by TaurusUY** [comment]
Hi Johnny, I downloaded the binary from the release following the tutorial with:

$ wget https://github.com/xetys/hetzner-kube/releases/download/0.0.3/hetzner-kube

**Comment by xetys** [comment]
hi. Yeah, you are using a really old version (0.0.3), while the current stable is 0.3 and the master the current state.

You could either download a binary from this page: https://github.com/xetys/hetzner-kube/releases/tag/0.3 

or build it from master. Build instructions can be found in our central README.


The reason why 0.0.3 is no longer working, as we don't use the install-docker-kubeadm shell script from the github repo anymore. Consequently this breaks the old version.

**Comment by TaurusUY** [comment]
**Hi xetys, thanks for that hint**. 
Now with the proper binary downloaded (0.3) cluster creation worked like a charm.

The only thing is failing now for me following the tutorial is the last step (HELM and INGRESS section) I am able to install Helm, with Tiller pod creation successfully, but when try to create via Helm the ingress it fails:

USER@kube-test-master-01:~$ helm install **--name ingress** --set rbac.create=true,controller.kind=DaemonSet,controller.service.type=ClusterIP
**Error: This command needs 1 argument: chart name**

So I tryed removing the "--name" arg and the error changed:

USER@kube-test-master-01:~$ helm install **ingress** --set rbac.create=true,controller.kind=DaemonSet,controller.service.type=ClusterIP
**Error: failed to download "ingress" (hint: running `helm repo update` may help)**

**So I did what was recommended by output error:**

USER@kube-test-master-01:~$ **helm repo update**
Hang tight while we grab the latest from your chart repositories...
...Skip local chart repository
**...Successfully got an update from the "stable" chart repository
Update Complete. ⎈ Happy Helming!⎈** 

Helm repo seems to be updated, but still it fails to create ingress.
When ran again, still got the same error message from HELM not been updated?

USER@kube-test-master-01:~$ helm install ingress --set rbac.create=true,controller.kind=DaemonSet,controller.service.type=ClusterIP
Error: failed to download "ingress" (hint: running `helm repo update` may help)

any other hint? :D
